app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
We need this?

Reply:
Let's keep it for now. I'd like to discuss views in this project once again.